src/app/api/v1/premium/route.ts

```diff
@@ -16,7 +16,7 @@ export async function GET(req: NextRequest) {
     const _email = searchParams.get("email");
 
     return NextResponse.redirect(
-      `${process.env.SERVER_URL}/user/breaches`,
+      `${process.env.SERVER_URL ?? ""}/user/breaches`,
```

Comment:
In what cases would `process.env.SERVER_URL` be missing/falsy?
I think we use it elsewhere without checking or using nullish coalescing for a fallback.

<details>
<summary><kbd>git grep -En "process\.env\.SERVER_URL"</kbd></summary>

```sh
src/app/(nextjs_migration)/(authenticated)/user/breaches/page.tsx:38:      url: process.env.SERVER_URL,
src/app/(nextjs_migration)/(guest)/breach-details/[breachName]/page.tsx:72:      url: process.env.SERVER_URL,
src/app/(nextjs_migration)/(guest)/breaches/page.tsx:26:      url: process.env.SERVER_URL,
src/app/api/v1/premium/route.ts:19:      `${process.env.SERVER_URL ?? ""}/user/breaches`,
src/app/api/v1/user/verify-email/route.ts:16:      (process.env.SERVER_URL ?? "") + "/user/settings",
src/app/layout.tsx:23:      typeof process.env.SERVER_URL === "string"
src/app/layout.tsx:24:        ? new URL(process.env.SERVER_URL)
src/app/layout.tsx:37:      url: process.env.SERVER_URL,
src/appConstants.js:69:if (!process.env.SERVER_URL && (process.env.NODE_ENV) === 'heroku') {
src/appConstants.js:70:  process.env.SERVER_URL = `https://${process.env.HEROKU_APP_NAME}.herokuapp.com`
```
</details>

Reply:
Heh, never, that env var is set in all our environments, but the lint rule doesn't want us to use potentially-undefined values in strings (which generally is a good rule-of-thumb).